core/tx_pool.go

```diff
@@ -324,7 +342,7 @@ func (pool *TxPool) loop() {
 	defer journal.Stop()
 
 	// Track the previous head headers for transaction reorgs
-	head := pool.chain.CurrentBlock()
+	head := pool.chain.CurrentBlock().Header()
```

Comment:
why make this change?

Reply:
Should be an oversight from some older refactoring from me. will remove it.